src/ansys/fluent/core/module_config.py

```diff
@@ -75,249 +78,250 @@ class Config:
+    launch_fluent_timeout = _ConfigDescriptor["Config"](
         lambda instance: int(instance._env.get("PYFLUENT_LAUNCH_FLUENT_TIMEOUT", 60))
     )
 
     #: Whether to show the Fluent GUI when launching the server, defaults to the value of ``PYFLUENT_SHOW_SERVER_GUI`` environment variable.
-    show_fluent_gui = _ConfigDescriptor(
+    show_fluent_gui = _ConfigDescriptor["Config"](
         lambda instance: instance._env.get("PYFLUENT_SHOW_SERVER_GUI") == "1"
     )
 
     #: Whether to launch Fluent in debug mode, defaults to the value of ``PYFLUENT_FLUENT_DEBUG`` environment variable.
-    fluent_debug = _ConfigDescriptor(
+    fluent_debug = _ConfigDescriptor["Config"](
         lambda instance: instance._env.get("PYFLUENT_FLUENT_DEBUG") == "1"
     )
 
     #: Whether to skip API upgrade advice, defaults to the value of ``PYFLUENT_SKIP_API_UPGRADE_ADVICE`` environment variable.
-    skip_api_upgrade_advice = _ConfigDescriptor(
+    skip_api_upgrade_advice = _ConfigDescriptor["Config"](
         lambda instance: instance._env.get("PYFLUENT_SKIP_API_UPGRADE_ADVICE") == "1"
     )
 
     #: The maximum number of bytes to log in gRPC logs, defaults to the value of ``PYFLUENT_GRPC_LOG_BYTES_LIMIT`` environment variable or 1000 bytes.
-    grpc_log_bytes_limit = _ConfigDescriptor(
+    grpc_log_bytes_limit = _ConfigDescriptor["Config"](
         lambda instance: int(instance._env.get("PYFLUENT_GRPC_LOG_BYTES_LIMIT", 1000))
     )
 
     #: Whether to disable the fix for returning parameter lists via settings API, defaults to the value of ``PYFLUENT_NO_FIX_PARAMETER_LIST_RETURN`` environment variable.
-    disable_parameter_list_return_fix = _ConfigDescriptor(
+    disable_parameter_list_return_fix = _ConfigDescriptor["Config"](
         lambda instance: instance._env.get("PYFLUENT_NO_FIX_PARAMETER_LIST_RETURN")
         == "1"
     )
 
     #: Whether to use runtime Python classes for settings, defaults to the value of ``PYFLUENT_USE_RUNTIME_PYTHON_CLASSES`` environment variable.
-    use_runtime_python_classes = _ConfigDescriptor(
+    use_runtime_python_classes = _ConfigDescriptor["Config"](
         lambda instance: instance._env.get("PYFLUENT_USE_RUNTIME_PYTHON_CLASSES") == "1"
     )
 
     #: Whether to hide sensitive information in logs, defaults to the value of ``PYFLUENT_HIDE_LOG_SECRETS`` environment variable.
-    hide_log_secrets = _ConfigDescriptor(
+    hide_log_secrets = _ConfigDescriptor["Config"](
         lambda instance: instance._env.get("PYFLUENT_HIDE_LOG_SECRETS") == "1"
     )
 
     #: The Fluent root directory to be used for PyFluent, defaults to the value of ``PYFLUENT_FLUENT_ROOT`` environment variable.
-    fluent_root = _ConfigDescriptor(
+    fluent_root = _ConfigDescriptor["Config"](
         lambda instance: instance._env.get("PYFLUENT_FLUENT_ROOT")
     )
 
-    #: The Ansys license path to be used in Fluent, defaults to the value of ``ANSYSLMD_LICENSE_FILE`` environment variable.
```

Comment:
Removed this from config to avoid accidentally printing it. User must control it only via environment variable.